relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -143,6 +146,7 @@ pub(crate) fn extract_span_tags(event: &mut Event, config: &Config) {
     let is_mobile = shared_tags
         .get(&SpanTagKey::Mobile)
         .is_some_and(|v| v.as_str() == "true");
+    let start_type = get_event_start_type(event.clone(), is_mobile);
```

Comment:
Would move this directly above `ttid` or `ttfd` (after the spans check right below).